aptos-move/framework/aptos-stdlib/sources/data_structures/smart_table.move

```diff
@@ -26,6 +28,8 @@ module aptos_std::smart_table {
     const EINVALID_TARGET_BUCKET_SIZE: u64 = 6;
     /// Invalid target bucket size.
     const EEXCEED_MAX_BUCKET_SIZE: u64 = 7;
+    /// Lengths of keys and values do not match
+    const EMISMATCHED_LENGTHS: u64 = 8;
```

Comment:
You added this here, but don't seem to use it.

Reply:
oh yeah, I deleted the usage. thanks